proto/storage/runtime_filters.proto

```diff
@@ -0,0 +1,34 @@
+syntax = "proto3";
+
+package storage;
+
+option cc_enable_arenas = true;
+option go_package = "storage";
+
+message RuntimeFilter {
+  enum RuntimeFilterFeatures {
+    EXTERNAL_IPS = 0;
+    PROCESSES = 1;
+  }
+
+  enum RuntimeFiltersType {
+    STRING = 0; // Exact match by comparing namespaces to the string directly.
+    REGEX = 1;  // Regex applied to namespaces in search of a match.
+  }
+
+  enum RuntimeFiltersEntity {
+    NAMESPACE = 0;
+    DEPLOYMENT = 1;
```

Comment:
```suggestion
    DEPLOYMENT = 1;
    CLUSTER = 2;
```

Reply:
We should be able to turn features on and off at the cluster level as well.

Reply:
The customer should be able to turn features on and off at the cluster level as well.